runtime/ops/command.rs

```diff
@@ -0,0 +1,359 @@
+    .borrow_mut()
+    .resource_table
+    .take::<CommandResource>(rid)?;
+  let mut command = Rc::try_unwrap(command_resource).ok().unwrap().0;
+  let output = command.output().await?;
+
+  Ok(CommandOutput {
+    status: output.status.into(),
+    stdout: Some(output.stdout.into()),
+    stderr: Some(output.stderr.into()),
+  })
+}
+
+#[derive(Serialize)]
+#[serde(rename_all = "camelCase")]
+struct Child {
+  rid: ResourceId,
+  pid: u32,
+  stdin_rid: Option<ResourceId>,
+  stdout_rid: Option<ResourceId>,
+  stderr_rid: Option<ResourceId>,
+}
+
+fn op_command_spawn(
+  state: &mut OpState,
+  rid: ResourceId,
+  args: CommandIoArgs,
+) -> Result<Child, AnyError> {
+  let command_resource = state.resource_table.take::<CommandResource>(rid)?;
+  let mut command = Rc::try_unwrap(command_resource).ok().unwrap().0;
+  handle_io_args(&mut command, args)?;
+
+  let mut child = command.spawn()?;
+  let pid = child.id().expect("Process ID should be set.");
+
+  let stdin_rid = child
+    .stdin
+    .take()
+    .map(|stdin| state.resource_table.add(ChildStdinResource::from(stdin)));
+
+  let stdout_rid = child
+    .stdout
+    .take()
+    .map(|stdout| state.resource_table.add(ChildStdoutResource::from(stdout)));
+
+  let stderr_rid = child
+    .stderr
+    .take()
+    .map(|stderr| state.resource_table.add(ChildStderrResource::from(stderr)));
+
+  let child_rid = state
+    .resource_table
+    .add(ChildResource(AsyncRefCell::new(child)));
+
+  Ok(Child {
+    rid: child_rid,
+    pid,
+    stdin_rid,
+    stdout_rid,
+    stderr_rid,
+  })
+}
+
+fn op_command_child_status(
+  state: &mut OpState,
+  rid: ResourceId,
+  _: (),
+) -> Result<Option<CommandStatus>, AnyError> {
+  let resource = state.resource_table.get::<ChildResource>(rid)?;
+  let mut child = RcRef::map(resource, |r| &r.0).try_borrow_mut().unwrap();
+  Ok(child.try_wait()?.map(|status| status.into()))
+}
+
+async fn op_command_child_wait(
+  state: Rc<RefCell<OpState>>,
+  rid: ResourceId,
+  stdin_rid: Option<ResourceId>,
+) -> Result<CommandStatus, AnyError> {
+  let resource = state
+    .borrow_mut()
+    .resource_table
+    .take::<ChildResource>(rid)?;
+  let mut child = RcRef::map(resource, |r| &r.0).borrow_mut().await;
+  if let Some(stdin_rid) = stdin_rid {
+    let stdin = state
+      .borrow_mut()
+      .resource_table
+      .take::<ChildStdinResource>(stdin_rid)?;
+    child.stdin = Some(Rc::try_unwrap(stdin).unwrap().into_inner());
+  }
+  Ok(child.wait().await?.into())
+}
+
+#[derive(Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct ChildStdio {
+  rid: ResourceId,
+  stdout_rid: Option<ResourceId>,
+  stderr_rid: Option<ResourceId>,
+}
+
+async fn op_command_child_output(
+  state: Rc<RefCell<OpState>>,
+  args: ChildStdio,
+  _: (),
+) -> Result<CommandOutput, AnyError> {
+  let resource = state
+    .borrow_mut()
+    .resource_table
+    .take::<ChildResource>(args.rid)?;
+  let resource = Rc::try_unwrap(resource).ok().unwrap();
```

Comment:
```suggestion
  let resource = Rc::try_unwrap(resource).unwrap();
```

Reply:
`try_unwrap` doesnt allow calling `unwrap` directly on it, so gotta hack around that with `ok`